src/app/models/share.ts

```diff
@@ -28,6 +29,9 @@ export default (database: Sequelize): ShareModel => {
         type: DataTypes.STRING,
         unique: true,
       },
+      mnemonic: {
+        type: DataTypes.BLOB,
```

Comment:
We currently have an issue with this field type on the user model as the value returned by the database is a Buffer but [is typed as a string](https://github.com/internxt/drive-server/blob/master/src/app/models/user.ts), is this the proper type for Sequelize?

Reply:
What is the issue?

Idk how it will react on fetch time, but storing works fine. I checked the type on the migrations and it was BLOB so i set it as BLOB. You mean when fetching there are problems?